lib/perl/Genome/Model/DifferentialExpression.pm

```diff
@@ -364,37 +367,37 @@ sub _resolve_workflow_for_build {
         right_operation => $differential_expression_operation,
         right_property => 'build_id'
     );
-    
+
     # Summarize Results
-    # TODO: cummerbund
     my $summarize_name = $self->summarize_differential_expression_name;
-    my $summarize_operation;
-    if ($summarize_name eq 'cummerbund') {
-        $summarize_operation = $workflow->add_operation(
-            name => 'Summarize Differential Expression',
-            operation_type => Workflow::OperationType::Command->create(
-                command_class_name => 'Genome::Model::DifferentialExpression::Command::Cummerbund',
-            )
+    if ($summarize_name) {
+        my $summarize_operation;
+        if ($summarize_name eq 'cummerbund') {
+            $summarize_operation = $workflow->add_operation(
+                name => 'Summarize Differential Expression',
+                operation_type => Workflow::OperationType::Command->create(
+                    command_class_name => 'Genome::Model::DifferentialExpression::Command::Cummerbund',
+                )
+            );
+        } else {
+            die('Unsupported summarize differential_expression_name: '. $summarize_name);
+        }
+        $summarize_operation->operation_type->lsf_queue($lsf_queue);
+        $summarize_operation->operation_type->lsf_project($lsf_project);
+        $workflow->add_link(
+            left_operation => $differential_expression_operation,
+            left_property => 'build_id',
+            right_operation => $summarize_operation,
+            right_property => 'build_id'
         );
-    } else {
-         die('Unsupported summarize differential_expression_name: '. $summarize_name);
-     }
+        $workflow->add_link(
+            left_operation => $summarize_operation,
+            left_property => 'result',
+            right_operation => $output_connector,
+            right_property => 'summarize_result'
+        );
+    }
     
```

Comment:
This would be better to live in it's own function, as adding additional summarizers will lead to less understandable code.